docs/en/building/basic-usage.md

```diff
@@ -0,0 +1,118 @@
+# Basic Usage
+
+## Creating a Dojo application
```

Comment:
Do you think we need to talk about creating an application in the build reference guide?

Reply:
Another requirement I'd originally listed in https://github.com/dojo/framework/issues/322. It's not strictly 'building' but is somewhat of a precursor requirement to it. Not sure where a better place would be for it - it feels too small to exist on its own. I'd say including it as a single section in this guide makes the most sense?

Reply:
That said, if we have plans to expand app creation to support more variations then it makes sense to separate it out into its own guide.

Reply:
Yeah I don't think the building reference guide should contain anything related to creating an application or cli-create-app - the assumption is you already have an application 😄 I do think there could/should be a small reference guide for using the "create" commands

Reply:
This was included so a user can go from absolutely nothing to building a dojo app using only this guide. Also to meet the "quickly scaffolding" requirement in the ticket. As a user there's value in a single guide that takes them through start to finish, even if this isn't the right place.

Reply:
I don't doubt there is value in guiding users through the create commands, but I don't think it should be at the beginning of of the build reference guide.